### PR TITLE
[DEV-3347] Paypal visual-dev updates

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

> Minor release with new features specifically added for the PayPal Payouts integration.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: [DEV-3347](https://saasquatch.atlassian.net/browse/DEV-3347)
- Process.st launch checklist: https://app.process.st/runs/Visual-Dev-120-PayPal-Admin-User-roles-piGSnsGUHa5OVaQ0UNpDIw/tasks/ipJPSwLUytFe3xMt6P5PYw

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [x] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [X] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [X] Changes have been reviewed by at least one other engineer
- [X] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/CHANGELOG.md [hunk 1]
```diff
@@ -12,17 +12,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Components changed
   - \<TagInput>
     - Added an input that supports tags before the functional input, can be used for tag or email input
```
Comment (DerekSiemens) [inline packages/visual-dev/CHANGELOG.md:14]: Change log in need of updating before launch

File: packages/visual-dev/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/visual-dev",
-  "version": "1.2.0-1",
```
Comment (DerekSiemens) [inline packages/visual-dev/package.json:3]: Flagging, needs updating before launch

File: packages/visual-dev/CHANGELOG.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.2.0] - 2022-11-
+## [1.2.0] - 2022-11-24
```
Comment (zachharrison) [inline packages/visual-dev/CHANGELOG.md:10]: Just calling this out now, we need to make sure we remember to tag all these release

File: packages/visual-dev/src/components/Input/rjsf-Input.tsx [hunk 4]
```diff
@@ -14,6 +14,7 @@ export function RJSFInput(props: WidgetProps) {
       disabled={props.disabled}
```
Comment (zachharrison) [inline packages/visual-dev/src/components/Input/rjsf-Input.tsx:14]: Would be nice if we could type these change events

File: packages/visual-dev/src/components/ObjectFieldTemplate/ObjectFieldTemplate.stories.tsx [hunk 5]
```diff
@@ -66,6 +66,65 @@ export const Default = () => {
   );
 };
 
+export const TwoColumns = () => {
```
Comment (zachharrison) [inline packages/visual-dev/src/components/ObjectFieldTemplate/ObjectFieldTemplate.stories.tsx:69]: Would be nice if we could add a bit of a gap here to make the visual look a bit better in storybook.

File: packages/visual-dev/src/components/Tag/Tag.tsx [hunk 6]
```diff
@@ -9,7 +9,7 @@ export interface OptionProps {
   /**
    * Callback triggered when the close on the tag is clicked
    */
-  onClickClose?: () => void;
+  onClickClose?: (e?: any) => void;
```
Comment (zachharrison) [inline packages/visual-dev/src/components/Tag/Tag.tsx:12]: Again, would nice to type these event

File: packages/visual-dev/src/components/Input/rjsf-Input.tsx (hunk 4 above)
Comment (noahwc) [reply packages/visual-dev/src/components/Input/rjsf-Input.tsx:14]: Agreed, unfortunately, RJSF doesn't type their parameters for `onChange` so I'm not sure we can artificially restrict type at this point, maybe @Locrian24 or @sam-beveridge can weight in.

**Comment by noahwc** [comment]
"Email Input" story name needs an update

**Review by zachharrison — APPROVED** [review]

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**